rom/laghos_timeinteg.cpp

```diff
@@ -42,6 +42,24 @@ void HydroODESolver::Init(TimeDependentOperator &_f)
     }
 }
 
+void HydroODESolver::SetSampler(ROM_Sampler *f)
+{
+    if (!rom)
```

Comment:
This function should be called only if `!rom`, meaning if not online. Would it be better to do `MFEM_VERIFY(rom)`?

Reply:
This routine has been removed as it is no longed needed.